engine/src/main/java/org/hibernate/search/query/dsl/sort/SortNativeContext.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.query.dsl.sort;
+
+/**
+ * @author Emmanuel Bernard emmanuel@hibernate.org
+ */
+public interface SortNativeContext extends SortAdditionalSortFieldContext, SortTermination {
```

Comment:
I guess the question is if you expect the existing "setSort" to be removed in future, and be replaced by this DSL as exclusive way?